src/views/v2/rsvp/form/fields/name.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Block: RSVP
+ * Form Name
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/form/fields/name.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ * @version TBD
+ */
+
+/**
+ * Set the default Full Name for the RSVP form
+ *
+ * @param string
+ * @param Tribe__Events_Gutenberg__Template $this
+ *
+ * @since 4.9
+ */
+$name = apply_filters( 'tribe_tickets_rsvp_form_full_name', '', $this );
```

Comment:
Why do we have these filters here not on the definition of the values?

Reply:
I don't understand what you mean.